src/terra-dev-site/contributing/ContributionGuidelines.g.contributing.mdx

```diff
@@ -0,0 +1,36 @@
+# Contribution Guidelines
+
+These guidelines are geared more towards medium and large component contributions.
```

Comment:
Can we quantify this better than "medium and large contributions"? Are we specifically talking about introducing new components here? MVBs to existing components? I guess I'd call those large. 

I'm not sure what I'd call medium (without minimizing contributions altogether)